src/transformers/modeling_tf_utils.py

```diff
@@ -318,8 +307,15 @@ def load_tf_weights(model, resolved_archive_file):
 
                         weight_value_tuples.append((symbolic_weight, array))
 
+                    symbolic_weights_names.add(symbolic_weight_name)
```

Comment:
This will add the `symbolic_weight_name` to `symbolic_weights_names` even if `saved_weight_value` is `None`. Is that expected?

Reply:
Yes, this is expected. I need to have the name of all the layers from the checkpoint on one side, and from the instantiated model on the other side, in order to properly compute the missing and unexpected layers.